visallo/visallo
# fix property visibility changes not displaying on the diff panel

Testing Instructions:
- Add a property without any visibility, make sure the diff panel show public visibility
- Modify the property visibility to A, make sure the diff panel shows visibility A
- Add a new property with visibility A, make sure the diff panel shows visibility A
- Modify the property visibility from A to "", make sure the diff panel shows public visibility

Points of Regression: N/A

NO CHANGELOG
